Fitbit.Portable.Tests/SleepTests.cs

```diff
@@ -14,6 +14,8 @@ namespace Fitbit.Portable.Tests
     [TestFixture]
     public class SleepTests
     {
+        #region Success Test
```

Comment:
@amammay Not a fan of the region personally. We probably need to look at refactoring the tests if the files are getting big